spec/views/hq/users/index_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe "hq/users/index.html.erb", type: :view do
+  it 'should display a valid user' do
+    assign(:users, [ FactoryGirl.build_stubbed(:user, user_name: 'User1', email: 'Email1@example.com') ] )
+    render 
+    expect(rendered).to match /User1/
+    expect(rendered).to match /Email1@example.com/
+  end
+
+  it 'should have a New User link' do
+    assign(:users, [])
+    render
+    expect(rendered).to have_link('New User', new_hq_user_path)
+  end
+
+  describe 'should display assigned active sponsors correctly' do
+
+    context 'no active sponsors' do
+      it 'displays 0 active sponsors' do
+        @users = [ instance_double(User, active_sponsors: [], user_name: 'usr', email: 'email@e.e') ]
+        render
+        expect(rendered).to have_selector('td', text: '0')
```

Comment:
I've had that problem before with `instance_double` and gave up trying to solve it. Ended up using regular `double` too.